fk-testimonial/src/actions/action.js

```diff
@@ -1,27 +1,32 @@
-export const SET_TYPE = "Set_Type";
-export const SET_URL = "Set_Url";
-export const SET_STATUS="Set_Status"
+export const SET_TYPE = "Set_Type"
+export const SET_URL = "Set_Url"
+export const SET_STATUS = "Set_Status"
+export const SET_SCREEN = "SET_SCREEN"
 
 export const setTestimonialUrl = (data) => {
-
-  return {
-    type: SET_URL,
-    payload: { url: data },
-  };
-};
+	return {
+		type: SET_URL,
+		payload: { url: data },
+	}
+}
 
 export const setTestimonialType = (data) => {
-    console.log("type in action",data)
-  return {
-    type: SET_TYPE,
-    payload: { type: data },
-  };
-};
+	return {
+		type: SET_TYPE,
+		payload: { type: data },
```

Comment:
instead of creating function use dispatch directly

Reply:
we'll change the dispatch approach entirely in refactoring